apps/site/assets/ts/stop/components/StopPageRedesign.tsx

```diff
@@ -20,11 +20,22 @@ import { Alert } from "../../__v3api";
 const isStopPageAlert = ({ effect }: Alert): boolean =>
   ["suspension", "stop_closure", "station_closure", "shuttle"].includes(effect);
 
+const FullwidthErrorMessage = (
```

Comment:
nit-pick: For consistency this should be a function returning a react component.  We don't really use names with capital first letters